### DISCUSSION TITLE
Deploy to Heroku button failed at configure environment

### DISCUSSION THREAD

**Discussion by shishircse06** [chaskiq/chaskiq#951] [Q&A]
Hi,
I was trying to deploy with Heroku button and it failed at configure environment step. It says 
`We tried to create heroku-postgresql:mini, but received an error from the add-on provider. Try the request again, or log a support ticket and include this message: Available versions are 12, 13, 14, 15 (the default is 15)`
Anyone faced this? Thanks.

**Comment by sukunatg** [comment]
Use latest version of postgres
